engine/runtime-integration-tests/src/test/java/org/enso/compiler/test/ExecCompilerTest.java

```diff
@@ -535,9 +536,9 @@ public void testPropertlyIdentifyNameOfJavaClassInError() throws Exception {
 
   // Conversion methods cannot be specified as private
   @Test
-  public void illegalPrivateConversion() throws Exception {
+  public void illegalPrivateConversion() {
     var module =
-        ctx.eval(
+        ctxRule.eval(
             LanguageInfo.ID,
             """
         type My_Type
```

Comment:
There is `assertThat(out.toString(), containsString(expectedErrMsg));` ten lines bellow. Such a statement would be replaced by:
```
assertThat(ctxRule.stdOut(), containsString(expectedErrMsg));
```